assignments/arrays.js

```diff
@@ -63,34 +63,47 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
```

Comment:
Awesome, very concise. If you wanted to make sure that this solution would work even if the index was no longer a reliable way to get the car with ID 33, you could find it by using:
```
let car33 = inventor.find(function(car) {
  return car.id === 33; 
})
console.log(`Car 33 is a ${car33.car_year} ${car33.car_make} ${car33.car_model}`)
```